monkestation/code/modules/power/rbmk/rbmk.dm

```diff
@@ -0,0 +1,1091 @@
+//For my sanity :))
+
+#define COOLANT_INPUT_GATE airs[1]
+#define MODERATOR_INPUT_GATE airs[2]
+#define COOLANT_OUTPUT_GATE airs[3]
+
+#define RBMK_TEMPERATURE_OPERATING 640 //Celsius	//Default: 640
+//At this point the entire ship is alerted to a meltdown. This may need altering
+#define RBMK_TEMPERATURE_CRITICAL 900 //Celsius		//Default: 800
+#define RBMK_TEMPERATURE_MELTDOWN 1000 //Celsius		//Default: 900
+
+//How many process()ing ticks the reactor can sustain without coolant before slowly taking damage
+#define RBMK_NO_COOLANT_TOLERANCE 5	//Default: 5
+
+#define RBMK_PRESSURE_OPERATING 1000 //PSI		//Default: 1000
+#define RBMK_PRESSURE_CRITICAL 6469.59 //PSI	//Default: 1469.59
+
+//No more criticality than N for now.
+#define RBMK_MAX_CRITICALITY 3		//Default: 3
+
+//To turn those KWs into something usable
+#define RBMK_POWER_FLAVOURISER 600		//Default: 8000 //I want some use out of turbines for power
+
+//Math. Lame.
+#define KPA_TO_PSI(A) (A/6.895)
```

Comment:
These could probably be put into an actual defines file

Reply:
Added to DEFINES/Maths.dm